test/forward.jl

```diff
@@ -242,9 +242,15 @@ end
 
 f = ODEFunction(fb, jac = jac_with_count, paramjac = paramjac)
 p = [1.5, 1.0, 3.0]
-absolutely_no_ad_sensealg = ForwardSensitivity(autodiff=false, autojacvec=false, autojacmat=false)
-prob = ODEForwardSensitivityProblem(f, [1.0; 1.0], (0.0, 10.0), p, absolutely_no_ad_sensealg)
+absolutely_no_ad_sensealg = ForwardSensitivity(autodiff = false,
+    autojacvec = false,
+    autojacmat = false)
+prob = ODEForwardSensitivityProblem(f,
+    [1.0; 1.0],
+    (0.0, 10.0),
+    p,
+    absolutely_no_ad_sensealg)
```

Comment:
Shit I didn't check this. This looks to be the wrong format.

Reply:
I just used julia formatter with the SciMLStyle